solenoid_app/views.py

```diff
@@ -178,23 +182,23 @@ def voltageChart(request):
                 x = 'x'
                 length = data['length'].magnitude + 1.0
                 for k, v in solenoid_range(data['voltage'], data['length'], data['r0'], data['ra'], data['awg'],
-                                           data['x'], data['force'], output_unit, data['xGraph'], idv_unit, 0.0, length,
+                                           data['x'], data['force'], output_unit, data['xGraph'], idv_unit, data['xStart'], length,
                                            1.0):
                     labels.append(k)
                     graph.append(round(v, sig_figs))
```

Comment:
When testing Joseph's PR earlier, I found a bug here. We have to account for when someone tries to solve x vs length where x is the independent.

Something like this should be fine:
x = 'x'
if data["length"] is None:
    length = data['length']
else:
    length = data['length'].magnitude + 1.0

Reply:
the bug is that when solving for length the value is none. So you cause an exception trying to get magnitude of a NoneType

Reply:
Hey @valdiston can we sync up about this? Wouldn't we want to inform the user that they can't have that as input?